### PR TITLE
feat: new searchCard design

### PR DESCRIPTION
**Pull request by bobetbat**
- Offers filtered and added to accommodation object
- new search card design 
- card now redirects on click and shows pin on map onHover

### CODE REVIEW

File: src/container/facility/FacilityOffers.tsx [hunk 1]
```diff
@@ -3,16 +3,14 @@ import { RoomCard } from 'src/components/RoomCard';
...
+  const { getAccommodationById, accommodations } = useAccommodationsAndOffers();
   const params = useParams();
   const id: string = params.id as string;
   const accommodation = getAccommodationById(accommodations, id);
-  const matchedOffers = getOffersById(offers, id);
 
   return (
     <>
-      {matchedOffers.map((offer, index) => {
+      {accommodation?.offers.map((offer, index) => {
```
Comment (kostysh) [inline src/container/facility/FacilityOffers.tsx:13]: is it possible to get accommodation without `offers`?

Comment (bobetbat) [reply src/container/facility/FacilityOffers.tsx:13]: nope ,It could be empty array.

Comment (kostysh) [reply src/container/facility/FacilityOffers.tsx:13]: if it `nope ,It could be empty array` you should not put question mark there

Comment (kostysh) [reply src/container/facility/FacilityOffers.tsx:13]: please use a question mark only as a `last chance`

Comment (bobetbat) [reply src/container/facility/FacilityOffers.tsx:13]: its for accommodation not to be `null`

Comment (bobetbat) [reply src/container/facility/FacilityOffers.tsx:13]: Screenshot 2022-08-22 at 13 55 34

**Review by kostysh — APPROVED** [review]
LGTM

**Pull request merged**